templates/main_template.yml

```diff
@@ -25,15 +25,13 @@ jobs:
 
       - name: Compile plugins
         run: |
-          mkdir plugins
-          cd scripting
-          spcomp -E -w234 -O2 -v2 -i"include" -o"../plugins/${plugin_name}.smx" ${plugin_name}.sp
+          mkdir ../plugins
+          spcomp -w234 -O2 -v2 -i"include" -o"../plugins/${plugin_name}.smx" ${plugin_name}.sp
```

Comment:
Whilst it'd be good practice to not have any warnings in your plugin especially when you're releasing them, it's not enforced by default ( neither by SPComp nor by the extension itself ).